src/index.tsx

```diff
@@ -358,6 +360,43 @@ function deleteSelectedElements() {
   }
 }
 
+function save() {
+  if (elements && elements.length > 0) {
```

Comment:
why are you checking for length > 0? It seems like the logic below also works for 0 elements. Also, elements should never be null/undefined